### PR TITLE
Adapt the dashboard to Loki

### PR DESCRIPTION
**Pull request by Kristian-ZH**
**What this PR does / why we need it**:
Adapt Gardener dashboard to Loki

This PR depends on this one: gardener/gardener#2515 and should wait until it is merged

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```noteworthy user
Kibana Access Information has been removed in this release. Use Grafana Access Information on the Monitoring Card to access logs
```

### CODE REVIEW

File: frontend/src/views/ShootDetails.vue [hunk 1]
```diff
@@ -99,12 +91,7 @@ export default {
     shootItem () {
       return get(this, 'value', {})
     },
-    isLoggingFeatureGateEnabled () {
-      const userCredentialsAvailable = !!this.info.logging_username && !!this.info.logging_password
-      const adminCredentialsAvailable = !!this.seedInfo.logging_username && !!this.seedInfo.logging_password
-      return userCredentialsAvailable || adminCredentialsAvailable
-    },
-    tickets () {
+    journals () {
```
Comment (grolu) [inline frontend/src/views/ShootDetails.vue:94]: this seems to be a merge issue
```suggestion
    tickets () {
```

Comment (grolu) [reply frontend/src/views/ShootDetails.vue:94]: @petersutter

**Comment by grolu** [comment]
lgtm, besides the merge issue. Need to wait for the gardener PR to be merged though. Therefore added the do-not-merge label

File: frontend/src/views/ShootDetails.vue (hunk 1 above)
Comment (Kristian-ZH) [reply frontend/src/views/ShootDetails.vue:94]: Yes, it is maybe a issue from the rebasing. Gonna fix it :)

Comment (petersutter) [reply frontend/src/views/ShootDetails.vue:94]: correct, @grolu

**Comment by petersutter** [comment]
@Kristian-ZH also one backend test is failing
```
         shoots
           should return shoot seed info:

      AssertionError: expected 2 to equal 4
```
could you please check?

**Comment by petersutter** [comment]
@Kristian-ZH as gardener/gardener#2515 is merged now we could also merge this PR.
Anything else you need to change in this PR or has nothing changed in the meanwhile?

**Pull request merged**